cas-server-support-openid/src/main/java/org/jasig/cas/support/openid/web/mvc/OpenIdValidateController.java

```diff
@@ -36,12 +78,14 @@ public void setValidationSpecificationClass(@Value("org.jasig.cas.validation.Cas
     @Autowired
     public void setFailureView(@Value("casOpenIdServiceFailureView") final String failureView) {
         super.setFailureView(failureView);
+        this.failureView = failureView;
     }
 
     @Override
     @Autowired
     public void setSuccessView(@Value("casOpenIdServiceSuccessView") final String successView) {
         super.setSuccessView(successView);
+        this.successView = successView;
```

Comment:
No, they're private to the parent. I can change them to protected or add a getter. Since there are already setters, i'm going to opt for getter since it makes it more 'bean-like'.